spec/controllers/controller_spec.rb

```diff
@@ -171,7 +171,7 @@
     # --- auto_login(user) ---
     specify { should respond_to(:auto_login) }
 
-    it 'auto_login(user) los in a user instance' do
+    it 'auto_login(user) logs in an user instance' do
```

Comment:
Shouldn't it be `a user instance` instead of `an user instance`? :)

Reply:
@mladenilic I fixed that! Thank you